snap/info.go

```diff
@@ -350,6 +351,22 @@ func (s *Info) ExpandSnapVariables(path string) string {
 	})
 }
 
+// InstallDate returns the "install date" of the snap.
+//
+// If the snap is not active, it'll return a zero time; otherwise
+// it'll return the modtime of the "current" symlink. Sneaky.
```

Comment:
IIrc (a long time ago) we had the idea to show two times: install-time and last-refresh-time. And iirc (again) this is why the top-level dir time was used. I wonder if we should clarify a bit more that this is the install data of this particular revision. Maybe something like: `InstallDate returns the "install date" of this snap revision` ? Alternatively we could have InstallData (the old semanitc) and LastRefreshDate but it would not make much sense to have this on a SnapInfo so I think this is all fine.

Reply:
but we're not looking at that timestamp, because it's occluded by the mount